scripts/linear-ticket/validate.py

```
@@ -0,0 +1,532 @@
+                                          critical=True):
+                return 1
+        return 0
+
+    def finish_exempt(self, headline: str, status_desc: str) -> int:
+        summary("## linear-ticket: ✅ exempt")
+        summary("")
+        summary(headline)
+        self.gh.delete_marker_comment(self.pr_number)
+        if self._guard_supersession():
+            if not self.gh.publish_status(self.validated_sha, "success", status_desc,
+                                          self.run_url, critical=True):
```

Ruff (0.16.1)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)